docs/PayPalNativeCheckout/README.md

```diff
@@ -0,0 +1,172 @@
+# Accepting PayPal Native Checkout Payments
+
+The PayPal Native Checkout module in the PayPal SDK enables PayPal payments via a webview in your app.
+
+Follow these steps to add PayPal Web Checkout payments:
+
+1. [Setup a PayPal Developer Account](#setup-a-paypal-developer-account)
+1. [Add PayPal Native Checkout Module](#add-paypal-native-checkout-module)
+1. [Test and go live](#test-and-go-live)
+
+## Setup a PayPal Developer Account
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID. 
+
+You will need a server integration to create an order to capture funds using the [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
+For initial setup, the `curl` commands below can be used as a reference for making server-side RESTful API calls.
+
+## Add PayPal Native Checkout Module
+
+### 1. Add the PayPal Native checkout to your app
+
+#### Swift Package Manager
+
+In Xcode, add the PayPal SDK as a [package dependency](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) to your Xcode project. Enter https://github.com/paypal/iOS-SDK as the package URL. Tick the "PayPalNativeCheckout" checkbox to add the PayPal Native Checkout package to your app.
+
+In your app's source code files, use the following import syntax to include the PayPal Native Checkout module:
+
+```swift
+import PayPalNativeCheckout
+```
+
+#### CocoaPods
+
+Include the PayPal Native Checkout pod in your `Podfile`.
+
+```ruby
+pod 'PayPalNativeCheckout'
+```
+
+In your app's source files, use the following import syntax to include PayPal's libraries:
+
+```swift
+import PayPalNativeCheckout
+```
```

Comment:
```suggestion
```

_Take it or leave it comment -_ I think we can omit the actual `import` statement instructions from the docs. Including instructions on how to include the module via SPM and CocoaPods should suffice for an iOS dev ([example](https://developer.paypal.com/braintree/docs/guides/apple-pay/client-side/ios/v5#custom-ui-integration) in Braintree developer docs).

Reply:
Sounds good!